### PR TITLE
Update polytope example docs with latest code and visualisations

### PR DESCRIPTION
**Pull request by awarde96**

### CODE REVIEW

File: docs/examples/polytope.ipynb [hunk 1]
```diff
@@ -54,146 +59,800 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c232c51b",
+   "metadata": {},
+   "source": [
+    "The collection being accessed is `ecmwf-mars`. The endpoint being accessed is `polytope.ecmwf.int`.\n",
+    "\n",
+    "To view the downloaded coveragejson you can run teh following cell."
```
Comment (sandorkertesz) [inline docs/examples/polytope.ipynb:103]: should be the

File: docs/examples/polytope.ipynb [hunk 2]
```diff
@@ -54,146 +59,800 @@
...
-    "import earthkit.data\n",
+    "ds.to_xarray()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f58a63b8",
+   "metadata": {},
+   "source": [
+    "## Visualise \n",
```
Comment (sandorkertesz) [inline docs/examples/polytope.ipynb:706]: This should be a chapter inside "Time series" but "Visualise"  is now on the same level.

File: docs/examples/polytope.ipynb [hunk 3]
```diff
@@ -54,146 +59,800 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "32cb0e10-7545-4758-b2f0-99984f01d71f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ds._json()"
```
Comment (sandorkertesz) [inline docs/examples/polytope.ipynb:113]: _json should not be advertised. If we need to expose it a public method should added in a separate PR.

Comment (jameshawkes) [reply docs/examples/polytope.ipynb:113]: does it make sense to expose it? I would think so, but not sure if this needs to be consistent with other data sources?

Comment (sandorkertesz) [reply docs/examples/polytope.ipynb:113]: Not sure what the method name should be and what it should return. A dict?

**Comment by sandorkertesz** [comment]
If we want to publish it without a release this should be branched off from "main" and then  merged into "main".

**Comment by sandorkertesz** [comment]
What happened with the original content of this example. Is it outdated now?

**Comment by jameshawkes** [comment]
You are right, maybe on this page we should just:
1. update the link to polytope readthedocs
2. add a sentence at the top saying that features can also be extracted, and link to the covjson examples

Sorry for the backtrack, @awarde96 could you do that?

**Comment by sandorkertesz** [comment]
The polytope links should also be updated in `sources.rst`.

**Pull request closed without merging**